.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          token: ${{ secrets.BOT_TOKEN }}
+          path: "pr-branch"
+
+      - name: Checkout charts repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ env.CHARTS_REPOSITORY }}
+          token: ${{ secrets.BOT_TOKEN }}
+          path: "charts-repo"
+
+      - name: Checkout development repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ env.DEVELOPMENT_REPOSITORY }}
+          token: ${{ secrets.BOT_TOKEN }}
+          path: "dev-repo"
+
+      - name: Set up Python 3.x Part 1
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+
+      - name: Set up Python scripts in pr-branch
+        working-directory: ./pr-branch
+        run: |
+          # set up python scripts
+          echo "set up python script in $PWD"
+          python3 -m venv ve1
+          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+          cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+      - name: Check if only release file in PR
+        id: check_version_in_PR
+        working-directory: ./pr-branch
+        run: |
+          # check if release file only is included in PR
+          ve1/bin/release-checker --api-url=${{ github.event.pull_request._links.self.href }}
+
+      - name: Set up Python scripts on main branch
+        if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+        run: |
+          # set up python scripts
+          echo "set up python script in $PWD"
+          python3 -m venv ve1
+          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+          cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+      - name: Check for restricted files and user permissiom
+        id: check_authorization
+        if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+        run: |
+          # check for a restricted file and, if found, check user has permissiom
+          ve1/bin/check-user --api-url=${{ github.event.pull_request._links.self.href }} --user=${{ github.event.pull_request.user.login }}
+
+      - name: Check if version updated
+        id: check_version_updated
+        if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+        run: |
+          # check if version file was changed
+          ve1/bin/release-checker --version=${{ steps.check_version_in_PR.outputs.PR_version }}
+
+      - name: Sync the repositories
+        id: sync_repositories
+        if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This  is a bit of a hangover from when I was trying to get it to work and copying everything Allen did, as we check out the repositories using BOT_TOKEN, si GITHUB_TOKEN would fail, so will remove it entirely.